docs/advanced/witness.md

```diff
@@ -0,0 +1,84 @@
+```
+
+You can see the `harvester-node-1` use less of resource to running as expection. You need to satisfy the three-node requirement, and the promote controller will promote the second and third nodes.
+
+## Workloads on the witness node
+
+The witness node only runs the must workload like etcd. Now the following list is the supported workloads on the witness node.
+- **harvester-node-manager**
+- **cloud-controller-manager**
+- **etcd**
+- **kube-proxy**
+- **rke2-canal**
+- **rke2-multus**
+
+## Upgrade the cluster with witness node
+
+When you upgrade the cluster with a witness node, you can upgrade the cluster as usual. But if you have a degraded volume, the upgrade might fail. You can check the following limitations for the volume robustness.
+
+## Limitations
+
+There were some limitations for the cluster with witness.
+
+- Volume robustness: The witness node does not store any volume data. It means we will miss one replica when the cluster has the witness node due to the Longhorn default replica number (3). The upgrade might failed by the degraded volume. It happens when the cluster only has three nodes, one of which is the witness node.
+
+  To resolve this, you should create a new StorageClass and decrease its replica number from 3 -> 2. Then, set this StorageClass as a default StorageClass.
+  ![new storageclass replica 2](/img/v1.3/advanced/new-storageclass-rep-2.png)
+  ![set to default](/img/v1.3/advanced/set-to-default-sc.png)
+
+  If you already create some volume with the default StorageClass, you can manually decrease the replica counts from 3 -> 2 on Longhorn UI -> **Volume Page** as a workaround. You can get the related info for how to redirect to Longhorn UI from [here](../troubleshooting/harvester/#access-embedded-rancher-and-longhorn-dashboards)
+  ![redirect-to-longhorn-volume-page](/img/v1.3/advanced/redirect-to-longhorn-vol-page.png)
+  ![update-replica-count-to-2](/img/v1.3/advanced/update-replica-2.png)
+
+  After, that the volume robustness will be healthy.
+
+
+## Known issues
+
+1. When you create a cluster with the witness node, the page of network config creation can not find any useable uplink with all nodes.
+![create network config with all nodes](/img/v1.3/advanced/create-policy-with-all-nodes.png)
+![no uplink](/img/v1.3/advanced/no-uplink.png)
+
+  You can choose nodes other than the witness node as a workaround. Then, you can successfully create the network config.
+  ![create network config with specific node](/img/v1.3/advanced/create-policy-with-specific-node.png)
+  ![get uplink](/img/v1.3/advanced/get-uplink.png)
+
+  You need to repeat this operation for every node except the witness node. The configuration for all nodes could be the same.
+
+  Related issue: [[BUG] Unable to select NIC to create network config when cluster contains witness node](https://github.com/harvester/harvester/issues/5325)
```

Comment:
```suggestion
## Known Issues

When creating a cluster with a witness node, the **Network Config: Create** screen on the Harvester UI is unable to identify any NICs that can be used with all nodes.

![create network config with all nodes](/img/v1.3/advanced/create-policy-with-all-nodes.png)
![no uplink](/img/v1.3/advanced/no-uplink.png)

The workaround is to select a non-witness node and then select a NIC that can be used with that specific node.

![create network config with specific node](/img/v1.3/advanced/create-policy-with-specific-node.png)
![get uplink](/img/v1.3/advanced/get-uplink.png)

Related issue: [[BUG] Unable to select NIC to create network config when cluster contains witness node](https://github.com/harvester/harvester/issues/5325)
```

Reply:
Hi @jillian-maroket, 

This sentence
```
You need to repeat this operation for every node except the witness node. The configuration for all nodes could be the same.
```

We want to inform the user that the workaround must be executed on every non-witness node. How could we describe it here? Thanks!